app/src/main/res/layout/fragment_relation_pager.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView android:layout_height="match_parent"
+    android:layout_width="match_parent"
+    xmlns:android="http://schemas.android.com/apk/res/android" >
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@arvindnegi1 this and everything inside the `ConstraintLayout` should be shifted to the left by 1 tab